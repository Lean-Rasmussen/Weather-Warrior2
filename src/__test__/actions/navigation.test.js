//this describes the test for the navigation bar.
//module import
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';

import NavBar from '../../components/navigationBar';

const navBar = shallow(<NavBar />);


it('Navbar renders', ()=> {
    expect(navBar).toMatchSnapshot();
})
