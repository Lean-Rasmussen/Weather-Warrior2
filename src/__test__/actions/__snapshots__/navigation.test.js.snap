// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <button
          className="nav-bar__home"
          id="Home"
          onClick={[Function]}
>
          Home
</button>,
        <div
          className="nav-bar__new-char"
          id="NewChar"
>
          New Char
</div>,
        <div
          className="nav-bar __battle"
          id="Battle"
>
          Battle
</div>,
      ],
      "className": "nav-bar",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Home",
          "className": "nav-bar__home",
          "id": "Home",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": "Home",
        "type": "button",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "New Char",
          "className": "nav-bar__new-char",
          "id": "NewChar",
        },
        "ref": null,
        "rendered": "New Char",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Battle",
          "className": "nav-bar __battle",
          "id": "Battle",
        },
        "ref": null,
        "rendered": "Battle",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <button
            className="nav-bar__home"
            id="Home"
            onClick={[Function]}
>
            Home
</button>,
          <div
            className="nav-bar__new-char"
            id="NewChar"
>
            New Char
</div>,
          <div
            className="nav-bar __battle"
            id="Battle"
>
            Battle
</div>,
        ],
        "className": "nav-bar",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Home",
            "className": "nav-bar__home",
            "id": "Home",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": "Home",
          "type": "button",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "New Char",
            "className": "nav-bar__new-char",
            "id": "NewChar",
          },
          "ref": null,
          "rendered": "New Char",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Battle",
            "className": "nav-bar __battle",
            "id": "Battle",
          },
          "ref": null,
          "rendered": "Battle",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
